authentication/account/worker/user_deactivation_worker.go

```diff
@@ -53,11 +55,21 @@ func (w *userDeactivationWorker) deactivateUsers() {
 		}, "error while notifying users to deactivate")
 		return
 	}
+
+	rescheduledDeactivation := time.Now().Add(w.rescheduleDelay)
+
 	for _, identity := range identities {
+		err := w.App.Identities().BumpDeactivationSchedule(w.Ctx, identity.ID, rescheduledDeactivation)
```

Comment:
Run it in a transaction.

Reply:
Done, by moving the business logic to a service function.